### ISSUE TITLE
crash

### ISSUE THREAD

**Issue by yxwu** [TimeForANinja/node-ytpl#65]
My app crashed because of ytpl, logs as below:

```
/app/node_modules/ytpl/lib/util.js:22
    title: exports.removeHtml(importantTxt.match(PLAYLIST_NAME_REGEXP)[1]),
                                                                      ^

TypeError: Cannot read property '1' of null
    at Object.exports.getGeneralInfo (/app/node_modules/ytpl/lib/util.js:22:71)
    at IncomingMessage.<anonymous> (/app/node_modules/ytpl/lib/firstpage.js:30:29)
    at IncomingMessage.emit (events.js:214:15)
    at endReadableNT (_stream_readable.js:1178:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Comment by naugtur** [comment]
same as https://github.com/TimeForANinja/node-ytpl/issues/60
There's already fixes for this. Just need maintainer to merge

**Comment by TimeForANinja** [comment]
fixed with #64
